README.md

```diff
@@ -5,6 +5,8 @@
 [![Documentation](https://readthedocs.org/projects/freqtrade/badge/)](https://www.freqtrade.io)
 [![Maintainability](https://api.codeclimate.com/v1/badges/5737e6d668200b7518ff/maintainability)](https://codeclimate.com/github/freqtrade/freqtrade/maintainability)
 
+## Description
+
```

Comment:
This part is unnecessary, adding an unnecessary header, moving the content lower - which is not desired.

The other section is fine as is - but please revert this part of the change.